digitaltwins/samples/service/javascript/get_digital_twin.js

```diff
@@ -3,32 +3,31 @@
 
 const IoTHubTokenCredentials = require('azure-iot-digitaltwins-service').IoTHubTokenCredentials;
 const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').DigitalTwinServiceClient;
-
-const connectionString = process.env.IOTHUB_CONNECTION_STRING;
-const deviceId = process.env.IOTHUB_DEVICE_ID;
+const { inspect } = require('util');
```

Comment:
>inspect [](start = 8, length = 7)

Learn something new everyday.

Reply:
Same here :)

---
In reply to: [450402230](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450402230) [](ancestors = 450402230)